src/platform.ts

```diff
@@ -93,8 +93,10 @@ export class ExampleHomebridgePlatform implements DynamicPlatformPlugin {
 
   async initializeThinqConfig() {
     const partialThinqConfig: PartialThinqConfig = {
-      countryCode: this.config.country_code,
-      languageCode: this.config.language_code,
+      // If a user installs via the homebridge UI, these values
+      // may not be guaranteed
+      countryCode: this.config.country_code || 'US',
+      languageCode: this.config.language_code || 'en-US',
```

Comment:
Could you check if there’s any other places `this.config.languageCode` is used? If so it might be good to make this a shared constant (same for country as well)

Reply:
I'm fairly certain I already checked this. Can you elaborate more on what should be a shared constant?

Reply:
Oh you're right, this is the only spot